fix(module-rate-limit-ts): flaky TTL test uses 1ms expiry, races on CI

## Problem

`templates/module-rate-limit-ts/skeleton/src/rate-limit/__tests__/memory-store.test.ts:101` — the "expires keys after TTL" test sets a value with `ttl=1` (the store's TTL is in **milliseconds**, not seconds, despite the test comment calling it "a very short TTL"), then expects an immediate `get` to return the value:

```ts
await store.set("ephemeral", "value", 1);
const immediate = await store.get("ephemeral");
expect(immediate).toBe("value"); // ← flakes on slow runners
```

Because both calls are `async`, the event loop gets to run between them. On a slow CI runner the 1ms expiry has already elapsed by the time `get` executes, and the test fails with:

```text
AssertionError: expected null to be 'value' // Object.is equality
```

Seen most recently in [PR #78 CI run](https://github.com/nanohype/nanohype/actions/runs/24854951332/job/72765210902). The failing run is otherwise green (catalog validation, markdown lint, other template skeletons all pass). The flake is independent of what #78 changed.

## Store behavior (for reference)

```ts
async set(key, value, ttl?) {
  const expiresAt = ttl ? Date.now() + ttl : null;
  data.set(key, { value, expiresAt });
}
```

So `ttl=1` means expiry at `Date.now() + 1ms`. One event-loop tick later, the key is already stale.

## Fix

Change the TTL in the test to something that comfortably beats one tick of event-loop work on any realistic runner, then extend the post-expiry wait accordingly. Suggested: TTL=50, post-set wait=75ms (still a fast test, no longer a race).

```ts
await store.set("ephemeral", "value", 50);

const immediate = await store.get("ephemeral");
expect(immediate).toBe("value");

await new Promise((resolve) => setTimeout(resolve, 75));
const expired = await store.get("ephemeral");
expect(expired).toBeNull();
```

Alternative: use `vi.useFakeTimers()` and `vi.advanceTimersByTime()` to drive expiry deterministically. Fine either way; the timer-based version keeps wall-clock work out of the test entirely.

## Scope

- Just the one test. No production code change needed — the store is correct, the test is over-aggressive.
- Related class of flake: #65 (z-score test, fixed in #68). Same pattern — overly tight timing assumptions in a test.

## Priority

Low. Reliably reproducible only on slow runners; local runs pass. But it blocks unrelated PRs when it trips, so worth getting off the backlog.

--- a/templates/module-rate-limit-ts/skeleton/src/rate-limit/__tests__/memory-store.test.ts
+++ b/templates/module-rate-limit-ts/skeleton/src/rate-limit/__tests__/memory-store.test.ts
@@ -99,15 +99,14 @@ describe("in-memory rate limit store", () => {
   });
 
   it("expires keys after TTL", async () => {
-    // Set with a very short TTL
-    await store.set("ephemeral", "value", 1);
+    // TTL is in milliseconds. Use a window wide enough to survive an
+    // awaited get() without racing against expiry on a slow runner.
+    await store.set("ephemeral", "value", 50);
 
-    // Should still exist immediately
     const immediate = await store.get("ephemeral");
     expect(immediate).toBe("value");
 
-    // Wait for TTL to expire
-    await new Promise((resolve) => setTimeout(resolve, 10));
+    await new Promise((resolve) => setTimeout(resolve, 75));
 
     const expired = await store.get("ephemeral");
     expect(expired).toBeNull();

